docs/gallery/general/plot_configurator.py

```diff
@@ -0,0 +1,112 @@
+"""
+How to Configure Term Validations
+=================================
+
+This is a user guide for how to curate and take advantage of configuration files in
+order to more easily validate terms within datasets or attributes.
+
+Introduction
+-------------
+Users will create a configuration YAML file that outlines the fields (within a neurodata type)
+they want to be validated against a set of allowed terms. 
+After creating the configuration file, users will need to load the
+configuration file with the :py:func:`~pynwb.load_type_config` method.
+With the configuration loaded, every instance of the neurodata
+types defined in the configuration file will have the respective fields wrapped with a
+:py:class:`~hdmf.term_set.TermSetWrapper`.
+This automatic wrapping is what provides the term validation for the field value.
+For greater control on which datasets and attributes are validated
+against which sets of allowed terms, use the 
+:py:class:`~hdmf.term_set.TermSetWrapper` on individual datasets and attributes instead.
+You can follow the 
+`TermSet tutorial in the HDMF documentation 
+<https://hdmf.readthedocs.io/en/stable/tutorials/plot_term_set.html#sphx-glr-tutorials-plot-term-set-py>`_
+for more information.
+
+To unload a configuration, simply call :py:func:`~pynwb.unload_type_config`.
+We also provide a helper method to see the configuration that has been loaded:
+:py:func:`~pynwb.get_loaded_type_config`
+
+
+How to make a Configuration File
+--------------------------------
+To see an example of a configuration file, please refer to ``nwb_config.yaml`` under ``src/config``.
```

Comment:
```suggestion
To see an example of a configuration file, please refer to ``nwb_config.yaml`` under ``src/config``.
```

This file is not accessible if the user pip installs pynwb, which will be the vast majority of users. Can you point to a github link to this file instead? Also can you move the file to `/example` or another directory not in `/src/`since we are not using it by default?

Reply:
Would this link exist now or after the merge? The link pointing to it now would be on a branch.

Reply:
The link would exist after the merge, but you can figure out what the path would be after merge and link to that.

Reply:
Wouldn't that fail check external link since it won't exist. currently, it already fails due to zarr, but in practice this wouldn't work?

Reply:
Yes, that would fail the external link check before merge. That's OK. The link would exist after merge, and we can confirm that it works and the link check passes after merge.